src/objects/npe01__Contacts_And_Orgs_Settings__c.object

```diff
@@ -59,6 +59,16 @@
         <type>Text</type>
         <unique>false</unique>
     </fields>
+    <fields>
+        <fullName>Enforce_Accounting_Data_Consistency__c</fullName>
+        <defaultValue>false</defaultValue>
+        <description>Enables rules and automation that enforce consistency in accounting data.</description>
```

Comment:
@KietTruongSFDO, @markross-sfdo - Were these reviewed by doc?

Reply:
@jbrandolinisf Yep, the texts here and the texts that will be switched to custom labels were reviewed by @markross-sfdo prior.